cli/runtimeflags/flags.go

```diff
@@ -29,11 +30,14 @@ func defaultMemMaxBytes() uint64 {
 
 type Flags struct {
 	// these memory limits should be based on a shared resource model
+	aggMemMax  auto.Bytes
 	sortMemMax auto.Bytes
 	fuseMemMax auto.Bytes
 }
 
 func (f *Flags) SetFlags(fs *flag.FlagSet) {
+	f.aggMemMax = auto.NewBytes(uint64(agg.MaxValueSize))
+	fs.Var(&f.aggMemMax, "aggmem", "maximum memory used by aggregate function values in MiB, MB, etc")
```

Comment:
The description made me think this was the max memory for an entire aggregate function (which is unbounded) rather than the max memory for an individual value.

Reply:
@mattnibs: How about "maximum memory used per aggregate function value" or "maximum memory used per aggregate function result"?

Reply:
Sorry I totally dropped the ball on this! I like "maximum memory used per aggregate function value".

Reply:
No worries!

Made this change in c6c2533dc6b21ba3a65ef12f351afa47ce5db66c.